### ISSUE TITLE
Can't install cryfs 0.11.1 on Monterey - Cmake did not find OpenMP.

### ISSUE THREAD

**Issue by hasandz** [cryfs/homebrew-tap#10]
Fresh monterey on Apple M1, I cannot install cryfs. I've also tried reinstalling libomp. 

```
brew install cryfs/tap/cryfs                                          
==> Downloading https://ghcr.io/v2/homebrew/core/libffi/manifests/3.4.2
Already downloaded: /Users/x/Library/Caches/Homebrew/downloads/1fcd5858590a42dc0fad3ce5012bdc2be712f542cdc9763c9289d2ec9efd4b61--libffi-3.4.2.bottle_manifest.json
==> Downloading https://ghcr.io/v2/homebrew/core/libffi/blobs/sha256:6a3605cff71
Already downloaded: /Users/x/Library/Caches/Homebrew/downloads/24d861b405e3f34950d1c1cb3ac9820b87a155860d4c001c2a9f86e8b986cc1c--libffi--3.4.2.arm64_monterey.bottle.tar.gz
==> Downloading https://ghcr.io/v2/homebrew/core/gdbm/manifests/1.22
Already downloaded: /Users/x/Library/Caches/Homebrew/downloads/e8b58066428135b278689a186abeaae951e5816cd8204e57b85c80b41adb13cc--gdbm-1.22.bottle_manifest.json
==> Downloading https://ghcr.io/v2/homebrew/core/gdbm/blobs/sha256:25cfaedb0fd8f
Already downloaded: /Users/x/Library/Caches/Homebrew/downloads/79cddbc71979a3749644a793e742326f5ae03b943ba6137c38540d4199c1409a--gdbm--1.22.arm64_monterey.bottle.tar.gz
==> Downloading https://ghcr.io/v2/homebrew/core/mpdecimal/manifests/2.5.1
Already downloaded: /Users/x/Library/Caches/Homebrew/downloads/f367c2ee08c56b88be0662703a8e4275f8657608a268c8c44e845154b0cea543--mpdecimal-2.5.1.bottle_manifest.json
==> Downloading https://ghcr.io/v2/homebrew/core/mpdecimal/blobs/sha256:726e8ec0
Already downloaded: /Users/x/Library/Caches/Homebrew/downloads/ffb0c2f865ae1dafbb01e2e7d9772cf80bfaa1d725ec77730d2c3523eef8c6d0--mpdecimal--2.5.1.arm64_monterey.bottle.tar.gz
==> Downloading https://ghcr.io/v2/homebrew/core/xz/manifests/5.2.5
Already downloaded: /Users/x/Library/Caches/Homebrew/downloads/9447dbb900ef03dd46277002d89d92c65920edee6c60ea22c33be6cf99d01dc6--xz-5.2.5.bottle_manifest.json
==> Downloading https://ghcr.io/v2/homebrew/core/xz/blobs/sha256:fcda3e81efe284f
Already downloaded: /Users/x/Library/Caches/Homebrew/downloads/64dbe71f9204854898c794b9e289f4f607c483ef4d18e627b2b910c60fbb32fe--xz--5.2.5.arm64_monterey.bottle.tar.gz
==> Downloading https://ghcr.io/v2/homebrew/core/python/3.10/manifests/3.10.1
Already downloaded: /Users/x/Library/Caches/Homebrew/downloads/ba44b4cfaec4b0996a8fc2b2def6f21371f8c859781b668a30ceb2e6f4bbf3be--python@3.10-3.10.1.bottle_manifest.json
==> Downloading https://ghcr.io/v2/homebrew/core/python/3.10/blobs/sha256:610ffb
Already downloaded: /Users/x/Library/Caches/Homebrew/downloads/9f33296799fb0f3d8dc0e02e5a3f0248cc3f930fecbe9d5484feebad2976e337--python@3.10--3.10.1.arm64_monterey.bottle.tar.gz
==> Downloading https://ghcr.io/v2/homebrew/core/six/manifests/1.16.0_2-1
Already downloaded: /Users/x/Library/Caches/Homebrew/downloads/da758f292f24fca4cc0e2176b5f4c778e5482fc66c144f637e4c3c23a7b6aaa6--six-1.16.0_2-1.bottle_manifest.json
==> Downloading https://ghcr.io/v2/homebrew/core/six/blobs/sha256:560f73cafaea61
Already downloaded: /Users/x/Library/Caches/Homebrew/downloads/b92bf7290f3bb55c674acf9d90729b6048e8c847703cdffc737e4e4eb9c59882--six--1.16.0_2.all.bottle.1.tar.gz
==> Downloading https://ghcr.io/v2/homebrew/core/conan/manifests/1.44.0
Already downloaded: /Users/x/Library/Caches/Homebrew/downloads/736afed90004c510df0840a9560cd25eaf4f76a869bf7d9e649e1c8a34cc5d71--conan-1.44.0.bottle_manifest.json
==> Downloading https://ghcr.io/v2/homebrew/core/conan/blobs/sha256:e886809cb3ae
Already downloaded: /Users/x/Library/Caches/Homebrew/downloads/4b351fd2db60b35b6457688d5b449e04f2942d6911e18336e04c78c623fefdd4--conan--1.44.0.arm64_monterey.bottle.tar.gz
==> Downloading https://ghcr.io/v2/homebrew/core/ninja/manifests/1.10.2_1
Already downloaded: /Users/x/Library/Caches/Homebrew/downloads/3d885b06ee8943649e1e4f3edebcca2003dc2e9e332202cc2a0b16ccce4f8cae--ninja-1.10.2_1.bottle_manifest.json
==> Downloading https://ghcr.io/v2/homebrew/core/ninja/blobs/sha256:8efac5a9c8b7
Already downloaded: /Users/x/Library/Caches/Homebrew/downloads/deb592b6fb7dd715b57a170608e055406d978bbc68c284ffa7ecef67acef899a--ninja--1.10.2_1.arm64_monterey.bottle.tar.gz
==> Downloading https://ghcr.io/v2/homebrew/core/pkg-config/manifests/0.29.2_3
Already downloaded: /Users/x/Library/Caches/Homebrew/downloads/ac691fc7ab8ecffba32a837e7197101d271474a3a84cfddcc30c9fd6763ab3c6--pkg-config-0.29.2_3.bottle_manifest.json
==> Downloading https://ghcr.io/v2/homebrew/core/pkg-config/blobs/sha256:2af9bce
Already downloaded: /Users/x/Library/Caches/Homebrew/downloads/733b30dce1aac09a672845fb0d3c4c5dd29f2f6ea9a1868688689f0acf21755f--pkg-config--0.29.2_3.arm64_monterey.bottle.tar.gz
==> Downloading https://github.com/cryfs/cryfs/releases/download/0.11.1/cryfs-0.
Already downloaded: /Users/x/Library/Caches/Homebrew/downloads/0715d38efe3e320b77523b92c338b566fa5648178219391e3b2c885a1d74477d--cryfs-0.11.1.tar.xz
==> Installing cryfs from cryfs/tap
==> Installing dependencies for cryfs/tap/cryfs: libffi, gdbm, mpdecimal, xz, python@3.10, six, conan, ninja and pkg-config
==> Installing cryfs/tap/cryfs dependency: libffi
==> Pouring libffi--3.4.2.arm64_monterey.bottle.tar.gz
🍺  /opt/homebrew/Cellar/libffi/3.4.2: 17 files, 673.3KB
==> Installing cryfs/tap/cryfs dependency: gdbm
==> Pouring gdbm--1.22.arm64_monterey.bottle.tar.gz
🍺  /opt/homebrew/Cellar/gdbm/1.22: 24 files, 1MB
==> Installing cryfs/tap/cryfs dependency: mpdecimal
==> Pouring mpdecimal--2.5.1.arm64_monterey.bottle.tar.gz
🍺  /opt/homebrew/Cellar/mpdecimal/2.5.1: 71 files, 2.2MB
==> Installing cryfs/tap/cryfs dependency: xz
==> Pouring xz--5.2.5.arm64_monterey.bottle.tar.gz
🍺  /opt/homebrew/Cellar/xz/5.2.5: 95 files, 1.4MB
==> Installing cryfs/tap/cryfs dependency: python@3.10
==> Pouring python@3.10--3.10.1.arm64_monterey.bottle.tar.gz
==> /opt/homebrew/Cellar/python@3.10/3.10.1/bin/python3 -m ensurepip
==> /opt/homebrew/Cellar/python@3.10/3.10.1/bin/python3 -m pip install -v --no-d
🍺  /opt/homebrew/Cellar/python@3.10/3.10.1: 3,134 files, 57.9MB
==> Installing cryfs/tap/cryfs dependency: six
==> Pouring six--1.16.0_2.all.bottle.1.tar.gz
🍺  /opt/homebrew/Cellar/six/1.16.0_2: 20 files, 122.3KB
==> Installing cryfs/tap/cryfs dependency: conan
==> Pouring conan--1.44.0.arm64_monterey.bottle.tar.gz
🍺  /opt/homebrew/Cellar/conan/1.44.0: 1,572 files, 18MB
==> Installing cryfs/tap/cryfs dependency: ninja
==> Pouring ninja--1.10.2_1.arm64_monterey.bottle.tar.gz
🍺  /opt/homebrew/Cellar/ninja/1.10.2_1: 10 files, 388KB
==> Installing cryfs/tap/cryfs dependency: pkg-config
==> Pouring pkg-config--0.29.2_3.arm64_monterey.bottle.tar.gz
🍺  /opt/homebrew/Cellar/pkg-config/0.29.2_3: 11 files, 676.4KB
==> Installing cryfs/tap/cryfs
==> cmake . -GNinja -DBUILD_TESTING=off
Last 15 lines from /Users/x/Library/Logs/Homebrew/cryfs/01.cmake:
-- Looking for pthread.h - found
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Success
-- Found Threads: TRUE
-- Could NOT find OpenMP_C (missing: OpenMP_C_FLAGS OpenMP_C_LIB_NAMES)
-- Could NOT find OpenMP_CXX (missing: OpenMP_CXX_FLAGS OpenMP_CXX_LIB_NAMES)
-- Could NOT find OpenMP (missing: OpenMP_C_FOUND OpenMP_CXX_FOUND)
CMake Error at vendor/cryptopp/vendor_cryptopp/CMakeLists.txt:1173 (message):
  OpenMP: Did not find OpenMP.  Build without USE_OPENMP if you want to allow
  this.


-- Configuring incomplete, errors occurred!
See also "/tmp/cryfs-20211230-25970-1fdr1s0/CMakeFiles/CMakeOutput.log".
See also "/tmp/cryfs-20211230-25970-1fdr1s0/CMakeFiles/CMakeError.log".

If reporting this issue please do so at (not Homebrew/brew or Homebrew/core):
  https://github.com/cryfs/homebrew-tap/issues

```

```
brew reinstall libomp                                                                                                    
==> Downloading https://ghcr.io/v2/homebrew/core/libomp/manifests/13.0.0
Already downloaded: /Users/x/Library/Caches/Homebrew/downloads/e5d1162de21152d4e8edcdcad88daf8290f53769dd8c7677d94025982d5594fc--libomp-13.0.0.bottle_manifest.json
==> Downloading https://ghcr.io/v2/homebrew/core/libomp/blobs/sha256:2a7253a4e9ff0ed48b1ea2cf98f953d17a28037ce3b9cbfb22c0c733918e22
Already downloaded: /Users/x/Library/Caches/Homebrew/downloads/23065afe04a7593ccae727b6178a684abb78c6f38179096dd70ed6ff50e8c4c2--libomp--13.0.0.arm64_monterey.bottle.tar.gz
==> Reinstalling libomp
==> Pouring libomp--13.0.0.arm64_monterey.bottle.tar.gz
🍺  /opt/homebrew/Cellar/libomp/13.0.0: 9 files, 1.6MB
==> Running `brew cleanup libomp`...
Disable this behaviour by setting HOMEBREW_NO_INSTALL_CLEANUP.
Hide these hints with HOMEBREW_NO_ENV_HINTS (see `man brew`).

```

**Comment by weatherfrog** [comment]
+1

**Comment by pblgomez** [comment]
😢  +1

**Comment by user419** [comment]
Hit the same issue with 0.11.2, got it to work with setting `sudo ln -s /opt/homebrew/opt /usr/local/opt`

**Comment by useraccessdenied** [comment]
> Hit the same issue with 0.11.2, got it to work with setting `sudo ln -s /opt/homebrew/opt /usr/local/opt`

Can't get it to work with this too. Any other solutions?

**Comment by vladmiller** [comment]
Any feedback on above?

`sudo ln -s /opt/homebrew/opt /usr/local/opt` produces a different error

```
==> cmake . -GNinja -DBUILD_TESTING=off
Last 15 lines from /Users/vlad/Library/Logs/Homebrew/cryfs/01.cmake:
-- Compiler: /opt/homebrew/Library/Homebrew/shims/mac/super/clang++
-- Compiler options:
-- Compiler definitions:
-- Build type: Release
-- Building version

CMake Error at src/gitversion/gitversion.cmake:13 (MESSAGE):
  Error running versioneer.  Return code is: No such file or directory, error
  message is:
Call Stack (most recent call first):
  src/gitversion/CMakeLists.txt:4 (get_git_version)```

Installing `gitversion` did not solve the issue.

**Comment by klarrio-dlamb** [comment]
Hi @vladmiller - put some comments in #12 re: the versioneer error.

**Comment by smessmer** [comment]
This is fixed on HEAD in https://github.com/cryfs/cryfs/commit/daea2fb7888e9adbb59d4b8e1d53e118010698a7

**Comment by smessmer** [comment]
This is fixed and CryFS 0.11.3 is released. A plain old brew install cryfs/tap/cryfs should work again.

**Comment by owais** [comment]
Looks like there is another issue in 0.11.3 on ARM macbooks. https://github.com/cryfs/homebrew-tap/issues/14
